src/nft/components/bag/BagRow.tsx

```diff
@@ -278,57 +278,56 @@ export const UnavailableAssetsHeaderRow = ({
 
   if (!assets || assets.length === 0) return null
 
+  const moreThanOneUnavailable = assets.length > 1
+
   return (
     <Column className={styles.unavailableAssetsContainer}>
-      {assets.length === 1 && (
-        <BagRow asset={assets[0]} usdPrice={usdPrice} removeAsset={() => undefined} grayscale isMobile={isMobile} />
-      )}
-      {assets.length > 1 && (
-        <Column>
-          <Row
-            justifyContent="space-between"
-            marginBottom={isOpen ? '12' : '0'}
-            cursor="pointer"
-            onClick={() => {
-              !didOpenUnavailableAssets && setDidOpenUnavailableAssets(true)
-              toggleOpen()
-            }}
-          >
-            <Row gap="12" color="textPrimary" className={bodySmall}>
-              {!isOpen && <UnavailableAssetsPreview assets={assets.slice(0, 5)} />}
-              No longer available
-            </Row>
-            <Row color="textSecondary">{isOpen ? <ChevronUpBagIcon /> : <ChevronDownBagIcon />}</Row>
-            {!didOpenUnavailableAssets && (
-              <Row
-                position="relative"
-                width="20"
-                height="20"
-                color="textPrimary"
-                justifyContent="center"
-                cursor="pointer"
-                onClick={clearUnavailableAssets}
-              >
-                <TimedLoader />
-                <CloseTimerIcon />
-              </Row>
-            )}
+      <Column>
+        <Row
+          justifyContent="space-between"
+          marginBottom={isOpen || !moreThanOneUnavailable ? '12' : '0'}
```

Comment:
nit: could extract this condition since it's used a couple times (including it's inverse on ln296)